pages/pipelines/defining_steps.md.erb

```diff
@@ -60,7 +60,7 @@ Starting your YAML with the `steps` object, you can add as many steps as you req
 
 Before getting started with a `pipeline.yml` file, you'll need to tell Buildkite where it will be able to find your steps.
 
-In the YAML steps editor, add the following YAML:
+In the YAML steps editor in Buildkite, add the following YAML:
```

Comment:
We don't do a good job of documenting how to talk about the Buildkite dashboard, but it is better than "in Buildkite"

Reply:
What do you suggest? "In the Buildkite dashboard"?